packages/api/controllers/grantCycle.js

```diff
@@ -33,7 +33,7 @@ const update = async (req, res) => {
     if (name) grant.name = name;
     if (openedOn) grant.openedOn = openedOn;
     if (closedOn) grant.closedOn = closedOn;
-    if (isActive) grant.isActive = isActive;
+    if (isActive != null) grant.isActive = isActive;
```

Comment:
Will this be `null` or `undefined` if left out of the req.body? 

Running in chrome terminal an object that does not have a specified key returns `undefined`. Unless the express -ody-parser does some magic to nullify things.

```javascript 
const o = {a: 1}
>> undefined
o.b
>> undefined
```

I think we can and should write a couple tests for this to verify the above hypothesis.

I guess secondly, everything that I just said doesn't matter because you are using a `==` which doesn't check types so therefore `null == undefined` will return truthy. 

I think that we should be in a habit of using `===` or `!==` for boolean comparison.